Created Metrics Settings Page (#10339)
getsentry/sentry-docs: docs/product/explore/metrics/metrics-settings.mdx

---
title: "Metrics Settings"
sidebar_order: 15

> description: "Learn about how you can manage your metrics and the associated tags directly in Sentry."

Reviewer: ```suggestion
description: "Learn how to manage your metrics and their associated tags directly in Sentry."
```

---

<Include name="feature-stage-beta-metrics.mdx" />

<Include name="metrics-api-change.mdx" />

> The Metrics page under your project settings allows you to see and manage the metrics emitted within this project all in one place. You can access it by navigating to your project settings and selecting *Metrics*.

Reviewer: ```suggestion
The Metrics settings page allows you to see and manage the metrics emitted within a project all in one place. You can access it by navigating to your project settings and selecting *Metrics* from the left sidebar.
```

![Metric Settings](img/metric-settings.png)

## Emitted Metrics

On the Metrics settings page you can see a list of active metrics being emitted for your project. For each metric, you can see:
- the type (counter, gauge, distribution or set)
- the unit (e.g. second, millisecond, bytes, etc)
- the [cardinality](/product/metrics/#cardinality)

> Users with Admin, Manager or Owner privileges within your organization also have the ability **disable a metric** by clicking on "Disable". This will block the further ingestion of a metric and make it inaccessible in Metrics, Alerts and Dashboards. This can be useful when a metric is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.

Reviewer: ```suggestion
Users with Admin, Manager or Owner [privileges](/organization/membership/) within your organization also have the ability **disable a metric** by clicking on "Disable". This will block the further ingestion of a metric and make it inaccessible in Metrics, Alerts and Dashboards. This can be useful when a metric is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.
```

> Users with Admin, Manager or Owner priviledges within your organization also have the ability **disable a metric** by clicking on "Disable". This will block the further ingestion of a metric and make it inaccessible in Metrics, Alerts and Dashboads. This can be useful when a metric is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.

Reviewer: ```suggestion
Users with Admin, Manager or Owner privileges within your organization also have the ability **disable a metric** by clicking on "Disable". This will block the further ingestion of a metric and make it inaccessible in Metrics, Alerts and Dashboards. This can be useful when a metric is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.
```

> Users with Admin, Manager or Owner [privileges](/organization/membership/) within your organization also have the ability **disable a metric** by clicking on "Disable". This will block the further ingestion of a metric and make it inaccessible in Metrics, Alerts and Dashboards. This can be useful when a metric is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.

Reviewer: ```suggestion
Team Admins and users with Manager or Owner [privileges](/organization/membership/) within your organization also have the ability **disable a metric** by clicking on "Disable". This will block the further ingestion of a metric and make it inaccessible in Metrics, Alerts and Dashboards. This can be useful when a metric is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.
```
Just want to double check that team/project admins have this privilege too

## Metric Details

> Clicking on a metric takes you to the *Metric Details* page. Here you can see further details about the individual metric such as:

Reviewer: ```suggestion
Clicking on a metric takes you to the *Metric Details* page. Here, you can see further details about the individual metric such as:
```

- the observed volume in the last 30 days
- the associated [tags](/product/metrics/#augmenting-metrics-with-tags)
- the location in your code where the metric is emitted

![Metric Details](img/metric-details.png)

> Users with Admin, Manager or Owner privileges within your organization also have the ability **disable a tag** by clicking on "Disable". This will block the further ingestion of that particular tag on this metric and make it inaccessible in Metrics, Alerts, and Dashboards. This can be useful when a tag is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.

Reviewer: ```suggestion
Users with Admin, Manager or Owner [privileges](/organization/membership/) within your organization also have the ability **disable a tag** by clicking on "Disable". This will block the further ingestion of that particular tag on this metric and make it inaccessible in Metrics, Alerts, and Dashboards. This can be useful when a tag is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.
```

> Users with Admin, Manager or Owner priviledges within your organization also have the ability **disable a tag** by clicking on "Disable". This will block the further ingestion of that particular tag on this metric and make it inaccessible in Metrics, Alerts and Dashboads. This can be useful when a tag is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.

Reviewer: ```suggestion
Users with Admin, Manager or Owner privileges within your organization also have the ability **disable a tag** by clicking on "Disable". This will block the further ingestion of that particular tag on this metric and make it inaccessible in Metrics, Alerts, and Dashboards. This can be useful when a tag is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.
```

> Users with Admin, Manager or Owner [privileges](/organization/membership/) within your organization also have the ability **disable a tag** by clicking on "Disable". This will block the further ingestion of that particular tag on this metric and make it inaccessible in Metrics, Alerts, and Dashboards. This can be useful when a tag is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.

Reviewer: ```suggestion
Team Admins and users with Manager or Owner [privileges](/organization/membership/) within your organization also have the ability **disable a tag** by clicking on "Disable". This will block the further ingestion of that particular tag on this metric and make it inaccessible in Metrics, Alerts, and Dashboards. This can be useful when a tag is no longer desired or has been misconfigured, but it's not immediately possible to stop emitting it on all clients.
```
Just want to double check that team/project admins have this privilege too

> The Metric Details page can also be accessed directly from the Metrics Explorer page, by clicking the 3 dot menu in your query.

Reviewer: ```suggestion
The Metric Details page can also be accessed directly from the Metrics Explorer page, by clicking the 3 dot menu icon next to your query.
```
